### ISSUE TITLE
Please update byte-buddy to newest version

### ISSUE THREAD

**Issue by marcinpz** [cronn/reflection-util#23]
Current version comes with old version of byte-buddy which makes problems with Mockito under Java 21:
```
Caused by: java.lang.IllegalArgumentException: Java 21 (65) is not supported by the current version of Byte Buddy which officially supports Java 20 (64) - update Byte Buddy or set net.bytebuddy.experimental as a VM property
```

**Comment by bwaldvogel** [comment]
I just released version `2.15.0` which bumps the ByteBuddy dependency.

**Comment by marcinpz** [comment]
Thank you!
